quickstart_docker/Dockerfile_quickstart

```diff
@@ -0,0 +1,33 @@
+ARG BASE_CONTAINER=yikunkero/spark:latest
+
+FROM $BASE_CONTAINER as spark
+
+LABEL maintainer="Andrew Bauman"
+
+FROM spark as delta
+
+USER root
+
+ARG DELTA_CORE_VERSION="2.1.0"
+
+RUN pip install --no-cache-dir delta-spark==${DELTA_CORE_VERSION} \

```

Comment:
Not a blocker, any chance we can also install `pip install deltalake` we can include the Delta Rust API Python Bindings?

Reply:
100%, should be an easy change /famous-last-words

Reply:
this is completed and tested

Reply:
this is done

Reply:
thanks!